pom.xml

```diff
@@ -237,6 +237,15 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!-- Disabled rules as we do not see value in them. -->
+      -Xep:JUnitClassModifiers:OFF
+      -Xep:JUnitMethodDeclaration:OFF
+      -Xep:JUnitValueSource:OFF
+      -Xep:LexicographicalAnnotationListing:OFF
+      -Xep:MethodReferenceUsage:OFF
+      -Xep:StaticImport:OFF
```

Comment:
```suggestion
      <!-- we prefer to qualify all static method calls with class name -->
      -Xep:StaticImport:OFF
```

Reply:
We do most of code review in phone browsers, no ide, so code must be explicit.